projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -245,11 +245,21 @@ async function getVersion(options) {
 			({version: currentVersion} = JSON.parse(
 				await readFileAsync('package.json', 'utf8')
 			));
+
+			if (!currentVersion && fs.existsSync('lerna.json')) {
+				({version: currentVersion} = JSON.parse(
+					await readFileAsync('lerna.json', 'utf8')
+				));
+			}
 		}
 		catch (_error) {
-			currentVersion = '';
+
+			// Ignore
+
```

Comment:
Any specific reason for the empty rows here? @izaera is it just SF things?

Reply:
Yeah, I believe we require line breaks before and after comments. I believe its a rule that comes from DXP

Reply:
Yep. It's the SF, that adds empty lines around comments.